### ISSUE TITLE
Opening the Editor throws multiple asserts in the Console

### ISSUE THREAD

**Issue by LB-TomaszSerwanski** [o3de/o3de#15590]
**Describe the bug**
Multiple asserts are displayed in the Console after launching the Editor. Example assert (for the others, please refer to the attached Editor.log file):
File:
 `D:/Git/o3de/Gems/Atom/RHI/DX12/Code/Source/RHI/CommandListBase.cpp(51)`
Condition:
```
(12296) 'void __cdecl AZ::DX12::CommandListBase::SetSamplePositions(const struct AZ::RHI::MultisampleState &)'
Custom sample positions are not supported on this device
```

Please refer to the attached video and editor.log file for more details.

**Steps to reproduce**
1. Launch the Editor.

**Expected behavior**
No asserts are displayed in the Console.

**Actual behavior**
An assert is displayed in the Console.

**Screenshots/Video**

https://user-images.githubusercontent.com/86953108/230365572-606dd856-3e81-4567-8def-85fc47f84f46.mp4

**Found in Branch**
Development

**Commit ID from [o3de/o3de](https://github.com/o3de/o3de) Repository**
e495f15

**Desktop/Device:**
 - Device: PC
 - OS: Windows
 - Version 10
 - CPU AMD Ryzen 5 3600
 - GPU Nvidia RTX 2060 SUPER
 - Memory 16GB

**Attachments**
[Editor.log](https://github.com/o3de/o3de/files/11170139/Editor.log)

**Comment by monroegm** [comment]
Asserts look like they have to do with RHI. Sending to SIG-Graphics-Audio

**Comment by moudgils** [comment]
Fixed here in stab branch and it should be integrated into dev very soon - https://github.com/o3de/o3de/pull/15628

**Comment by LB-TomaszSerwanski** [comment]
Issue verified as fixed in the latest Development (1820553) and Stabilization/2305 (1ff603a) branches.
